### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by DevJac**
I've updated the dependencies and added a few `expect()` calls because the `room()` API changed.

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -1,19 +1,34 @@
-use std::collections::HashSet;
-
-use log::*;
-use screeps::{find, prelude::*, Part, ResourceType, ReturnCode, RoomObjectProperties};
+//use std::collections::HashSet;
+//use screeps::{find, prelude::*, Part, ResourceType, ReturnCode, RoomObjectProperties};
```
Comment (ASalvail) [inline src/main.rs:2]: Can you clean that up if it's no longer necessary?

File: src/main.rs [hunk 2]
```diff
@@ -22,113 +37,153 @@ fn main() {
...
-    Ok(())
+fn game_loop() {
+    debug!(
+        "Game Loop. CPU: {:5.2} / {}",
+        screeps::game::cpu::get_used(),
+        screeps::game::cpu::limit()
+    );
 }
+
+//debug!("running spawns");
```
Comment (ASalvail) [inline src/main.rs:86]: A lot of dead code.

**Review by ASalvail — CHANGES_REQUESTED** [review]
I think I understand what you were going for here, but by removing everything related to the basic script, it completely removes the starting point that was useful for newcomer in rust.

I believe the idea was to imitate the tutorial for screeps in JS, but in a rust format.

Also, I thought that `rand` didn't work with `wasm`!

**Comment by DevJac** [comment]
Thank you for taking a look.
I thought most of those changes were just personal, I didn't realize they would be affecting this PR.
All I ever intended to have merged was the first commit. I have fixed the PR.

**Comment by Eadword** [comment]
Can someone approve this PR. Seems like a pretty basic change that is one of the first things I had to fix after getting started myself. I was about to raise a PR only to notice someone had me beat :smile:

**Review by ASalvail — APPROVED** [review]

**Comment by ASalvail** [comment]
Sorry, I missed that. Thanks for the nudge @Eadword

**Pull request merged**